src/admin_module/user_module/director.py

```diff
@@ -0,0 +1,18 @@
+__author__ = 'alex alvarez'
+
+from person import Person
+
+
+class Director(Person):
+    def __init__(self, first_name, last_name):
+        Person.__init__(self, first_name, last_name)
+        self._filmography = []
+
+    def add_movie_to_filmography(self, movie):
+        self._filmography.append(movie)
+
+    def get_filmography_list(self):
+        return self._filmographhy
+
+    def directed_the_film(self, movie):
```

Comment:
Remember that method names should represent actions, in this case does not looks like an action is more like an stament, please change it to something more accurate